.travis.yml

```diff
@@ -18,6 +18,8 @@ install:
   - cd ..
   - docker cp db-test-seed.sql cacophony-api:/db-seed.sql
 before_script:
+  - sudo mkdir /etc/cacophony
+  - sudo chmod 777 /etc/cacophony
```

Comment:
Does this mean that tests are looking in /etc/cacophony? That's not going to be reliable, especially on developer machines. See below for more on this.